ground-server/src/app/controls/page.tsx

```diff
@@ -10,38 +10,69 @@ export default async function Controls() {
 
   return (
     <div>
-      <div className="grid grid-cols-2 gap-5 p-5">
-        <ActuationBox 
-          title="Quick Disconnect Motor" 
-          onButtonLabel="Connect" 
-          offButtonLabel="Disconnect"
+      <div className="grid grid-cols-1 lg:grid-cols-2 gap-5 p-5">
+        <ActuationBox
+          title="Quick Disconnect Motor"
+          onButtonLabel="Connect"
           onStateLabel="Connected"
-          offStateLabel="Disconnected" 
-          initialStateLabel="No Data" 
+          onCommand={{
+            qdRetract: false,
+          }}
+          offButtonLabel="Disconnect"
+          offStateLabel="Disconnected"
+          offCommand={{
+            qdRetract: true,
+          }}
+          initialStateLabel="No Data"
         />
-        <ActuationBox 
-          title="Ball Valve" 
-          onButtonLabel="Open" 
-          offButtonLabel="Close" 
+
+        <ActuationBox
+          title="Ball Valve"
+          onButtonLabel="Open"
           onStateLabel="Opened"
+          onCommand={{
+            bv1Open: true,
```

Comment:
These should be two separate actuation boxes

Reply:
Did this for now. Eventually I wanna expand on Aly's design so that we can have more than 2 (or less than 2) buttons in an actuation box.